src/Sarif.Multitool.Library/Rules/ADO1014.ProvideRequiredRunProperties.cs

```diff
@@ -13,6 +13,15 @@ public class AdoProvideRequiredRunPropertties
         /// </summary>
         public override string Id => RuleId.ADOProvideRequiredRunProperties;
 
+        protected override IEnumerable<string> MessageResourceNames => new string[] {
+            nameof(RuleResources.ADO1014_AdoProvideRequiredRunPropertties_Error_MissingAutomationDetails_Text),
+            nameof(RuleResources.ADO1014_AdoProvideRequiredRunPropertties_Error_MissingAutomationDetailsId_Text),
+            nameof(RuleResources.ADO1014_AdoProvideRequiredRunPropertties_Error_MissingToolDriver_Text),
+            nameof(RuleResources.ADO1014_AdoProvideRequiredRunPropertties_Error_MissingTool_Text),
```

Comment:
for example, we have 4 messages here,
below I see only 2 of them used.
Are there some functionality not yet implemented?

Reply:
Sorry, this is an old copy/paste. You should be able to remove it

Reply:
deleted as your input.